sos/plugins/scsi.py

```diff
@@ -32,4 +33,9 @@ def setup(self):
             "sg_map -x"
         ])
 
+        if os.path.isdir("/sys/class/scsi_host"):
+            for host in os.listdir("/sys/class/scsi_host"):
+                host_path = os.path.join('/sys/class/scsi_host/', host)
+                self.add_udev_info(host_path, attrs=True)
+
```

Comment:
I think this would look better using a glob: that way, you can get rid of the explicit presene test, and you don't have to do any path mangling on the directory content. We can also lose the explicit list walk since the `add_udev_info()` method supports list arguments:

```python
    hosts = glob.glob("/sys/class/scsi_host/*")
    self.add_udev_unfo(hosts, attrs=True)
```